src/odb/src/zutil/util.cpp

```diff
@@ -47,8 +49,35 @@ using std::vector;
 RUDYCalculator::RUDYCalculator(dbBlock* block) : block_(block)
 {
   gridBlock_ = block_->getDieArea();
+  // from RUDY paper: "The wire width p is defined by the
+  //  average wire-to-wire pitch ~p and number of routing layers l: p = ~p/l.
   wireWidth_ = block_->getTech()->findRoutingLayer(1)->getWidth();
-  makeGrid();
+
+  int track_spacing = 0;
+  odb::dbTechLayer* tech_layer = block_->getTech()->findRoutingLayer(3);
+  odb::dbTrackGrid* track_grid = block_->findTrackGrid(tech_layer);
+
+  // This code is the same on GRT, make a method for it? For example:
```

Comment:
You can move this implementation to the odb::dbTechLayer class and just call it here.